README.md

```diff
@@ -1,5 +1,7 @@
 [![TypeDB](./docs/banner.png)](https://typedb.com/introduction)
 
+<div align="center">
```

Comment:
Can we get rid of this? It's ugly when you read the `md` file in plain text, which is the most common use case for us developers. I think them not being centered is okay right?